specification.yaml

```diff
@@ -26,16 +26,37 @@ paths:
               schema:
                 type: string
   /{type}:
+    parameters:
+      - name: type
+        in: path
+        required: true
+        description: the type of ontology
+        schema:
+          type: string
+        example: examples
+    post:
+      security:
+        - ApiKeyAuth: []
+      description: Upload files to server
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                ontology-rdf-file:
```

Comment:
Kan man ikke bare laste opp 1 fil isteden for at man må bruke object med disse properties?

Reply:
Det kan man. Begge disse er optional.